### PR TITLE
Add Surfline to SQLFluff in the Wild

### PR DESCRIPTION
**Pull request by GClunies**
Linting brah! 🤙🏽  🏄🏽

### CODE REVIEW

**Review by barrywhart — APPROVED** [review]
🎉 💯

File: docs/source/inthewild.rst [hunk 1]
```diff
@@ -28,3 +28,15 @@ Just add a section below by raising a PR on Github by
 - `Petal's <https://www.petalcard.com>`_ data-eng team runs SQLFluff on our 100+ model
   `dbt <http://www.getdbt.com/>`_ project. As a pre-commit hook and as a CI
   check, SQLFluff helps keep our SQL readable and consistent.
+- `Surline <https://www.surfline.com/>`_'s Analytics Engineering team
```
Comment (NiallRees) [inline docs/source/inthewild.rst:31]: Just a typo here 😁

**Review by NiallRees — CHANGES_REQUESTED** [review]
Surline - steaks directly to your door?

File: docs/source/inthewild.rst (hunk 1 above)
Comment (GClunies) [reply docs/source/inthewild.rst:31]: Well done! 🤣  🐮

**Comment by GClunies** [comment]
Well done @NiallRees! 🤣  🐮

**Review by NiallRees — APPROVED** [review]

**Pull request merged**